<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="Lenovo" name="com.test.OMRBranchClub" tests="4" failures="0" timestamp="14 Jul 2022 17:54:35 GMT" time="8.062" errors="1">
  <testcase name="updateAddress" time="0.000" classname="com.test.OMRBranchClub">
    <skipped/>
  </testcase> <!-- updateAddress -->
  <testcase name="basicAuthLogin" time="8.062" classname="com.test.OMRBranchClub">
    <error type="java.lang.RuntimeException" message="com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field &quot;exception&quot; (class com.pojo.Login_Output_pojo), not marked as ignorable (6 known properties: &quot;cart_count&quot;, &quot;status&quot;, &quot;role&quot;, &quot;message&quot;, &quot;data&quot;, &quot;refer_msg&quot;])
 at [Source: (String)&quot;{
    &quot;message&quot;: &quot;SQLSTATE[HY000] [1040] Too many connections (SQL: select * from `g_bus_rule_ref` where `rule_name` = currency limit 1)&quot;,
    &quot;exception&quot;: &quot;Illuminate\\Database\\QueryException&quot;,
    &quot;file&quot;: &quot;/home/omrbranch/public_html/vendor/laravel/framework/src/Illuminate/Database/Connection.php&quot;,
    &quot;line&quot;: 664,
    &quot;trace&quot;: [
        {
            &quot;file&quot;: &quot;/home/omrbranch/public_html/vendor/laravel/framework/src/Illuminate/Database/Connection.php&quot;,
            &quot;line&quot;: 624,
            &quot;fu&quot;[truncated 12817 chars]; line: 3, column: 19] (through reference chain: com.pojo.Login_Output_pojo[&quot;exception&quot;])">
      <![CDATA[java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "exception" (class com.pojo.Login_Output_pojo), not marked as ignorable (6 known properties: "cart_count", "status", "role", "message", "data", "refer_msg"])
 at [Source: (String)"{
    "message": "SQLSTATE[HY000] [1040] Too many connections (SQL: select * from `g_bus_rule_ref` where `rule_name` = currency limit 1)",
    "exception": "Illuminate\\Database\\QueryException",
    "file": "/home/omrbranch/public_html/vendor/laravel/framework/src/Illuminate/Database/Connection.php",
    "line": 664,
    "trace": [
        {
            "file": "/home/omrbranch/public_html/vendor/laravel/framework/src/Illuminate/Database/Connection.php",
            "line": 624,
            "fu"[truncated 12817 chars]; line: 3, column: 19] (through reference chain: com.pojo.Login_Output_pojo["exception"])
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:30)
at io.restassured.path.json.mapping.JsonPathObjectDeserializer$deserialize.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:46)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
at io.restassured.internal.mapping.Jackson2Mapper.deserialize(Jackson2Mapper.groovy:58)
at io.restassured.mapper.ObjectMapper$deserialize.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:46)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
at io.restassured.internal.mapping.ObjectMapping.parseWithJackson2(ObjectMapping.groovy:254)
at io.restassured.internal.mapping.ObjectMapping$parseWithJackson2$0.callStatic(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:54)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:217)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:240)
at io.restassured.internal.mapping.ObjectMapping.deserialize(ObjectMapping.groovy:56)
at io.restassured.internal.mapping.ObjectMapping$deserialize.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:46)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
at io.restassured.internal.RestAssuredResponseOptionsGroovyImpl.as(RestAssuredResponseOptionsGroovyImpl.groovy:231)
at io.restassured.internal.RestAssuredResponseOptionsImpl.as(RestAssuredResponseOptionsImpl.java:169)
at com.test.OMRBranchClub.basicAuthLogin(OMRBranchClub.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "exception" (class com.pojo.Login_Output_pojo), not marked as ignorable (6 known properties: "cart_count", "status", "role", "message", "data", "refer_msg"])
 at [Source: (String)"{
    "message": "SQLSTATE[HY000] [1040] Too many connections (SQL: select * from `g_bus_rule_ref` where `rule_name` = currency limit 1)",
    "exception": "Illuminate\\Database\\QueryException",
    "file": "/home/omrbranch/public_html/vendor/laravel/framework/src/Illuminate/Database/Connection.php",
    "line": 664,
    "trace": [
        {
            "file": "/home/omrbranch/public_html/vendor/laravel/framework/src/Illuminate/Database/Connection.php",
            "line": 624,
            "fu"[truncated 12817 chars]; line: 3, column: 19] (through reference chain: com.pojo.Login_Output_pojo["exception"])
at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1127)
at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2023)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1700)
at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1678)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:319)
at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:176)
at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:323)
at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4674)
at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3629)
at io.restassured.internal.path.json.mapping.JsonPathJackson2ObjectDeserializer.deserialize(JsonPathJackson2ObjectDeserializer.java:28)
... 46 more
]]>
    </error>
  </testcase> <!-- basicAuthLogin -->
  <testcase name="addAddress" time="0.000" classname="com.test.OMRBranchClub">
    <skipped/>
  </testcase> <!-- addAddress -->
  <testcase name="deleteAddress" time="0.000" classname="com.test.OMRBranchClub">
    <skipped/>
  </testcase> <!-- deleteAddress -->
</testsuite> <!-- com.test.OMRBranchClub -->
